ui/perfherder/perf-helpers/helpers.js

```diff
@@ -888,3 +889,60 @@ export const getResultsMap = (results) => {
   });
   return { testNames, names, platforms, resultsMap };
 };
+
+export const getTimeDifference = (currentDate, dueDate) => {
+  const timeDifference = Math.abs(dueDate - currentDate);
+  let differenceInDays = Math.ceil(timeDifference / (1000 * 60 * 60 * 24));
+  // saturday and sunday are considered weekend days
+  const weekendDaysCount = 2;
+  // convert days to weeks and keeps the count of full weeks
+  const weeksCount = Math.trunc(differenceInDays / 7);
+  // count the total weekend days
+  const weekendDaysToSubstract = weekendDaysCount * weeksCount;
+
+  // [1. Mon] [2. Tue] [3. Wed] [4. Thu]  [5. Fri] [6. Sat] [7. Sun]
+  //                            startDate -------- -------- ----->>>
+  // -------- -------> endDate
+  if (currentDate.getUTCDay() > dueDate.getUTCDay()) {
+    // If weekday of startDate is bigger than endDate then it is a new week,
+    // and we have to subtract the weekend days
+    differenceInDays -= weekendDaysCount;
+  } else {
+    // substracts the total weekend days from the total days
+    differenceInDays -= weekendDaysToSubstract;
+  }
+
+  let hoursDifference = Math.ceil(timeDifference / (1000 * 60 * 60));
+  const shouldGetHoursLeft =
+    currentDate.getUTCDay() === weekdays.friday &&
+    dueDate.getUTCDay() === weekdays.monday;
+  // If due date is Monday and today is Friday and we have to show the hours left,
+  // we have to subtract the weekend hours from the difference
+  if (shouldGetHoursLeft) {
+    hoursDifference -= 2 * 24;
+  }
+  return {
+    hours: hoursDifference,
+    days: differenceInDays,
+  };
+};
+
+export const getCountdownText = (now, dueDate, difference) => {
+  if (difference.hours < 24 && difference.hours >= 0) {
+    return `${difference.hours} hours left`;
+  }
+
+  if (now.getTime() >= dueDate.getTime()) {
+    return `Overdue`;
+  }
+
+  return `${difference.days} days left`;
+};
+
+export const alertIsTriaged = (alertSummary) => {
```

Comment:
```suggestion
export const getAlertFirstTriagedDate = (alertSummary) => {
```
Otherwise your function should return a boolean based on the naming of it (usually a function that contains `is` or `has` expects to return a boolean).